apps/charterafrica/src/components/Infographic/Infographic.js

```diff
@@ -2,13 +2,15 @@ import { Box } from "@mui/material";
 import Script from "next/script";
 import React from "react";
 
+import site from "@/charterafrica/utils/site";
+
 const Infographic = React.forwardRef(function Infographic(props, ref) {
   const {
     countriesFile: countriesFileProp,
     dataFile: dataFileProp,
     infographicId: id,
     sx,
-    url,
+    url = site.url,
```

Comment:
If the component is aware of `site`, then there is no point of passing `url` as prop. It can access `site` object directly.
```suggestion
```

i.e. this is why the original plan was to process `url` in the backend.